tests/func/test_metrics.py

```diff
@@ -101,6 +131,26 @@ def test_show(self):
         self.assertSequenceEqual(ret["bar"]["metric_hcsv"], ["bar"])
         self.assertSequenceEqual(ret["baz"]["metric_hcsv"], ["baz"])
 
+        ret = self.dvc.metrics.show(
+            "metric_json_ext",
+            typ="json",
+            xpath="$.metrics[?(@.deviation_mse<0.30) & (@.value_mse>0.4)]",
```

Comment:
Wow, this is truly something :slightly_smiling_face: We should definitely document this. Could you please adjust `--xpath` description for json in https://dvc.org/doc/commands-reference/metrics-modify ?